src/store/actions/transactions.test.js

```diff
@@ -1,21 +1,48 @@
-import { actionTypes } from '@constants';
+import { actionTypes, loginTypes } from '@constants';
 import * as transactionsApi from '@api/transaction';
+import * as transactionsModifyApi from '@utils/transaction';
+import * as hwManagerApi from '@utils/hwManager';
 import {
+  emptyTransactionsData,
   transactionsRetrieved,
+  resetTransactionResult,
+  pendingTransactionAdded,
+  transactionDoubleSigned,
+  transactionBroadcasted,
+  transactionCreated,
 } from './transactions';
+import { sampleTransaction, transformedAccountTransaction, newTransaction } from '../../../test/constants/transactions';
 
 jest.mock('@api/transaction');
 jest.mock('@api/delegate');
+jest.mock('@utils/transaction');
+jest.mock('@utils/hwManager');
 
 describe('actions: transactions', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
   const dispatch = jest.fn();
   const getState = () => ({
```

Comment:
In the future we can add test utils to return default state or generic getState function for reusability

Reply:
Done